compiler/damlc/tests/BUILD.bazel

```diff
@@ -199,54 +202,71 @@ da_haskell_library(
             ":integration-lib",
         ],
     )
-    for version, skip_validation in [(version, False) for version in COMPILER_LF_VERSIONS] + [
+    for version, skip_validation, eval_order in [(version, False, "LeftToRight") for version in COMPILER_LF_VERSIONS] + [
         (
             lf_version_configuration.get("default"),
             True,
+            "LeftToRight",
         ),
         (
             lf_version_configuration.get("dev"),
             True,
+            "LeftToRight",
+        ),
+        (
+            lf_version_configuration.get("dev"),
+            False,
+            "RightToLeft",
```

Comment:
Would it also make sense to run the dev tests in righ-to-left order with skip_validation=True? I don't know what skip_validation is for.

Reply:
validation are all the package validation in the engine (typechecking ...), You can have a look at the scala package `com.daml.lf.validation`. 

We use this to avoid type checking in the case of the IDE, to gain a bit of reactivity (in this case we trust the compiler to generate valid LF). 
 
So to answer your question. No.